README.md

```diff
@@ -1,2 +1,174 @@
-# Budget-app
-It is a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what
+<a name="readme-top"></a>
+    <div align="center">
+
+  <img src="https://i.imgur.com/iwXJcq2.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Budget App</b></h3>
+</div>
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Recipe-app ](#-practice-test-driven-developmenttdd-)
+  - [🛠 Built With ](#-built-with-)
+  - [🛠 Tech Stack ](#-tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+  - [👥 Authors](#-authors)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Budget App <a name="about-project"></a>
+
+Budget app is an app for managing transactions with categorized lists, sign-up and login pages, and navigation buttons for moving between pages. Users can add new categories or transactions.
+
+## 🛠 Built With <a name="built-with"></a>
+
+> <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+> <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+> <li><a href="https://www.postgresql.org/">Postgresql</a></li>
+
+### Key Features <a name="key-features"></a>
+
+> - Add Categories.
+> - List Categories.
+> - Add Transacions.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://budget-br2s.onrender.com/)- Powered by Render
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Video Presentation <a name="presentation"></a>
+
+- [Video Presentation Link](https://www.loom.com/share/1735305b039a438685040b0b00894bc0)- Powered by Loom
```

Comment:
- [ ] Good Job with the video presentation. The only concern I have is discussing a piece of code/feature that you liked. it would be great if you can address that as well as per the project requirements.